.github/workflows/build.yml

```diff
@@ -0,0 +1,44 @@
+name: Build
+
+on:
+  push:
+    branches-ignore:
+      - master
+      - "1.16"
```

Comment:
workflows should be updated to remove references to 1.16

Reply:
I'll bump to 1.17 since that's coming up